erpnext/www/all-products/index.html

```diff
@@ -48,10 +48,10 @@ <h1>{{ _('Products') }}</h1>
 	<div class="col-12 order-2 col-md-8 order-md-1 products-list row no-gutters">
 		{% if products %}
 			{% for item in products %}
-				{% include "erpnext/www/all-products/item_row.html" %}
+				{% include "bloomstack_core/www/all-products/item_row.html" %}
```

Comment:
Why are you putting bloomstack_core reference in erpnext app? this change is not required. If only the path is changed and everything else is the same, then you don't have to raise a PR for this. Close this PR.